### PR TITLE
Add content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

### PR DESCRIPTION
**Pull request by guizanelato**
Added pt-br translation for content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md .

Preview: https://deploy-preview-26636--kubernetes-io-master-staging.netlify.app/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig/

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by jailton** [comment]
Welcome @guizanelato!

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title:  Organizando o acesso ao cluster usando arquivos kubeconfig
+content_type: conceito
```
Comment (jailton) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:3]: We don't need to translate the content-type. The correct is *concept*.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+```
+kubectl config use-context
+```
+
+## A variável de ambiente KUBECONFIG 
+
+A variável de ambiente `KUBECONFIG` possui uma lista dos arquivos kubeconfig. Para Linux e Mac, esta lista é delimitada por vírgula. No Windows, a lista é delimitada por ponto e vírgula. A variável de ambiente `KUBECONFIG` não é um requisito obrigatório - caso ela não existir o `kubectl` utilizará o arquivo kubeconfig padrão localizado no caminho `$HOME/.kube/config`.
+
+Se a variável de ambiente `KUBECONFIG` existir, o `kubeclt` utilizará uma configuração que é o resultado da combinação dos arquivos listados na variável de ambiente `KUBECONFIG`.
```
Comment (edsoncelio) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:52]: Just a small change: `kubeclt` to `kubectl`

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 1 above)
Comment (guizanelato) [reply content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: thanks for the info. already updated on 9f795b4

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+   Construir cada peça de informação do cluster baseado nas opções à seguir; a primeira ocorrência encontrada será a opção vencedora: 
+
+   1. Usar as flags de linha de comando caso existirem: `--server`, `--certificate-authority`, `--insecure-skip-tls-verify`.
+	 1. Se algum atributo do cluster existir a partir da combinação de kubeconfigs, estes deverão ser utilizados.
+   1. Se não existir informação de localização do servidor falhar. 
+
+1. Determinar a informação atual de usuário a ser utilizada. Construir a informação de usuário utilizando as mesmas regras utilizadas para o caso de informações de cluster, exceto para a regra de técnica de autenticação que deverá ser única por usuário:
+   
+	 1. Usar as flags, caso existirem: `--client-certificate`, `--client-key`, `--username`, `--password`, `--token`.
+   1. Usar os campos `user` resultado da combinação de arquivos kubeconfig.
```
Comment (bryanasdev000) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:109]: Extra spaces here.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+## Contexto
+
+Um elemento de *contexto* em um kubeconfig é utilizado para agrupar parâmetros de acesso em um nome conveniente. Cada contexto possui três parâmetros: cluster, namespace, e usuário.
+
+Por padrão, a ferramenta de linha de comando `kubectl` utiliza os parâmetros do _contexto atual_ para se comunicar com o cluster.
+
+Para escolher o contexto atual:
+
+```
```
Comment (bryanasdev000) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:44]: Missing "shell" here (same on EN).

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+Como descrito anteriormente, a saída poderá ser resultado de um único arquivo kubeconfig, ou poderá ser o resultado da junção de vários arquivos kubeconfig. 
+
+Aqui estão as regras que `kubectl` utiliza quando realiza a combinação de arquivos kubeconfig:
+
+1. Se o argumento `--kubeconfig` está definido, apenas o arquivo especificado será utilizado. Apenas uma instância desta flag é permitida.
+
+   Caso contrário, se a variável de ambiente `KUBECONFIG` estiver definida, esta deverá ser utilizada como uma lista de arquivos a serem combinados.
+	 Combine os arquivos listados na variável de ambiente `KUBECONFIG` de acordo com as seguintes regras:
+
+   * Ignorar arquivos vazios.
```
Comment (bryanasdev000) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:71]: Extra space.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+	 1. Se o contexto não estiver vazio, utilizar o cluster ou usuário deste contexto.
+
+   O usuário e o cluster poderá estar vazio neste ponto.
+
+1. Determinar as informações do cluster atual a serem utilizadas. Neste ponto, poderá ou não existir informações de um cluster.
+
+   Construir cada peça de informação do cluster baseado nas opções à seguir; a primeira ocorrência encontrada será a opção vencedora: 
+
+   1. Usar as flags de linha de comando caso existirem: `--server`, `--certificate-authority`, `--insecure-skip-tls-verify`.
+	 1. Se algum atributo do cluster existir a partir da combinação de kubeconfigs, estes deverão ser utilizados.
```
Comment (bryanasdev000) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:103]: Extra/Missing space.

**Review by bryanasdev000 — COMMENTED** [review]
It has small cosmetic changes that can affect when rendering (I haven't tested it) but the translation looks good to me.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 6 above)
Comment (guizanelato) [reply content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: added to bdf0282.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 5 above)
Comment (guizanelato) [reply content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: added to bdf0282.

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md (hunk 4 above)
Comment (guizanelato) [reply content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md]: added to bdf0282.

**Review by bryanasdev000 — APPROVED** [review]
LGTM

**Comment by jailton** [comment]
LGTM

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+Para escolher o contexto atual:
+
+```shell
+kubectl config use-context
+```
+
+## A variável de ambiente KUBECONFIG 
+
+A variável de ambiente `KUBECONFIG` possui uma lista dos arquivos kubeconfig. Para Linux e Mac, esta lista é delimitada por vírgula. No Windows, a lista é delimitada por ponto e vírgula. A variável de ambiente `KUBECONFIG` não é um requisito obrigatório - caso ela não existir o `kubectl` utilizará o arquivo kubeconfig padrão localizado no caminho `$HOME/.kube/config`.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:50]: "caso ela não exista"

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+Para inspecionar a sua configuração atual, execute o seguinte comando:
+
+```shell
+kubectl config view
+```
+
+Como descrito anteriormente, a saída poderá ser resultado de um único arquivo kubeconfig, ou poderá ser o resultado da junção de vários arquivos kubeconfig. 
+
+Aqui estão as regras que `kubectl` utiliza quando realiza a combinação de arquivos kubeconfig:
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:64]: NIT: "que o `kubectl`"

Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:64]: NIT: "que o `kubectl`"

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+1. Se o argumento `--kubeconfig` está definido, apenas o arquivo especificado será utilizado. Apenas uma instância desta flag é permitida.
+
+   Caso contrário, se a variável de ambiente `KUBECONFIG` estiver definida, esta deverá ser utilizada como uma lista de arquivos a serem combinados.
+   Combine os arquivos listados na variável de ambiente `KUBECONFIG` de acordo com as seguintes regras:
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:69]: IMO here is a bit disconnected from the previous statement, maybe "[...] a serem combinados, seguindo o seguinte fluxo" and ignore the next statement :)

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+    * Ignorar arquivos vazios.
+    * Produzir erros para aquivos cujo conteúdo não for possível desserializar. 
+    * O primeiro arquivo que definir um valor ou mapear uma chave determinada, será o escolhido.
+    * Nunca modificar um valor ou mapear uma chave.
+	   Exemplo: Preservar o contexto do primeiro arquivo que definir `current-context`.
+		 Exemplo: Se dois arquivos especificar um `red-user`, use apenas os valores do primeiro `red user`. Mesmo se um segundo arquivo possuir entradas não conflitantes sobre a mesma entrada `red-user`, estas deverão ser descartadas.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:76]: "se dois arquivos especificarem"

Comment (rikatz) [reply content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:76]: red-user vs "red user" here (maybe the second red user in this statement should also be red-user)

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+   Um contexto vazio é permitido neste momento.
+
+
+1. Determinar o cluster e o usuário. Neste ponto, poderá ou não existir um contexto.
+   Determinar o cluster e o usuário no primeiro padrão encontrado de acordo com a ordem à seguir. Este procedimento deverá executado duas vezes: uma para definir o usuário a outra para definir o cluster.
+
+	1. Utilizar a flag caso ela existir: `--user` ou `--cluster`.
+	1. Se o contexto não estiver vazio, utilizar o cluster ou usuário deste contexto.
+
+   O usuário e o cluster poderá estar vazio neste ponto.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:96]: poderão estar vazios

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+
+   Construir cada peça de informação do cluster baseado nas opções à seguir; a primeira ocorrência encontrada será a opção vencedora: 
+
+  1. Usar as flags de linha de comando caso existirem: `--server`, `--certificate-authority`, `--insecure-skip-tls-verify`.
+  1. Se algum atributo do cluster existir a partir da combinação de kubeconfigs, estes deverão ser utilizados.
+  1. Se não existir informação de localização do servidor falhar. 
+
+1. Determinar a informação atual de usuário a ser utilizada. Construir a informação de usuário utilizando as mesmas regras utilizadas para o caso de informações de cluster, exceto para a regra de técnica de autenticação que deverá ser única por usuário:
+   
+	1. Usar as flags, caso existirem: `--client-certificate`, `--client-key`, `--username`, `--password`, `--token`.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:108]: missing space here, this is a subitem from the previous item (see the preview vs https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/, items are numbered incorrectly

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+	1. Usar os campos `user` resultado da combinação de arquivos kubeconfig.
+	1. Se existirem duas técnicas conflitantes, falhar.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:110]: missing space here, this is a subitem from the previous item (see the preview vs https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/, items are numbered incorrectly

File: content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+
+   O usuário e o cluster poderá estar vazio neste ponto.
+
+1. Determinar as informações do cluster atual a serem utilizadas. Neste ponto, poderá ou não existir informações de um cluster.
+
+   Construir cada peça de informação do cluster baseado nas opções à seguir; a primeira ocorrência encontrada será a opção vencedora: 
+
+  1. Usar as flags de linha de comando caso existirem: `--server`, `--certificate-authority`, `--insecure-skip-tls-verify`.
+  1. Se algum atributo do cluster existir a partir da combinação de kubeconfigs, estes deverão ser utilizados.
+  1. Se não existir informação de localização do servidor falhar.
```
Comment (rikatz) [inline content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md:104]: missing space here, this is a subitem from the previous item (see the preview vs https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/, items are numbered incorrectly

**Review by jhonmike — APPROVED** [review]

**Pull request merged**